addons/html_editor/static/src/editor/list/list_plugin.js

```diff
@@ -247,13 +247,14 @@ function indent(liElement) {
 // returns: is still in a <LI> nested list
 // former oShiftTab
 function outdent(liElement) {
+    const doc = liElement.ownerDocument;
```

Comment:
Why not use the document from the plugin ?

Reply:
Because this is a helper function, not a plugin method.
(should it become a method?)